docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1187,6 +1194,41 @@ structured_annotations {
 }
 ~ End Proto
 
+### `SourceLocation` Message
+
+A source location describes a location within a *.p4*-source file. The
+`SourceLocation` message is defined in p4types.proto as follows:
+
+~ Begin Proto
+// Location of code relative to a given source file.
+message SourceLocation {
+  // Path to the source file (total or relative to the working directory).
```

Comment:
I think this is normally called absolute, not total.

Reply:
Here and below.

Reply:
Good catch. Done.